libs/article/website/src/lib/article-container.stories.tsx

```diff
@@ -2297,7 +2297,6 @@ export default {
 
 export const ById = {
   args: {
-    onQuery: action('onQuery'),
```

Comment:
removed all `onQuery` as they didn't work as intended, idea was to move to render props in the future, but I don't know if that even makes sense. One can get all that information by using apollo in cache only fetch mode.